docs/src/user/datasets.rst

```diff
@@ -83,3 +83,29 @@ An example of ``corpus.tsv`` for POS tagging follows:
     presidents  8   
     .           4
     \n          0
+
+
+.. _`dataset-type:TABULAR`:
+
+TABULAR
+--------------------------------------------------------------------
+
+The dataset file must be of the ``.zip`` archive format with a ``table_meta.txt`` and the tabular dataset of the ``.csv`` format at the root of the directory.
```

Comment:
Thinking of changing `table_meta.txt` to `table_meta.json` since `.json` is a more specific file format. Let me know if there's any issues with this

Reply:
I still prefer kwargs. 
table_meta.json adds one more file to be managed (uploaded and versioned).
or you may want to add a model.yaml for every model and put all the configuration in that file including the training, validation, test path, etc. yaml file is preferred than json file.

Reply:
Based on our framework, attributes for model training such as training/validation dataset files and dataset metadata (which is this) would need to be provided by the *application developer* (not the *model developer*) during the creation of the train job, and this metadata would be forwarded to every model to-be-trained.

Following your feedback, I'm thinking of adding an additional field to the [`create_train_job()` method](https://nginyc.github.io/rafiki/docs/latest/src/user/quickstart-app-developers.html#creating-a-train-job through), either with a `kwargs` field or a YAML metadata file. Do you think that is okay?

Reply:
let's add kwargs first.
if we have many arguments to add later, we can add the yaml file.

Reply:
Working on this